overlays/custom-packages/waypipe/default.nix

```diff
@@ -5,9 +5,7 @@
 prev.waypipe.overrideAttrs (_prevAttrs: {
   # Upstream pull request: https://gitlab.freedesktop.org/mstoeckl/waypipe/-/merge_requests/21
   patches = [
-    ./waypipe-fix-reading-data-from-pipes.patch
     ./waypipe-window-borders.patch
-    ./waypipe-security-context.patch
   ];
   nativeBuildInputs = _prevAttrs.nativeBuildInputs ++ [ prev.pkgs.wayland-scanner ];
   buildInputs = _prevAttrs.buildInputs ++ [ prev.pkgs.wayland ];
```

Comment:
I believe the nativeBuildInputs and buildInputs lines can also be deleted as they were required for the security context patch which is now in the upstream